ui/templatetags/render_bundle.py

```diff
@@ -12,20 +12,23 @@
 
 register = template.Library()
 
+def public_path(request):
+    if settings.DEBUG and settings.USE_WEBPACK_DEV_SERVER:
+        return webpack_dev_server_url(request)
+    else:
+        return static("bundles/")
+
 
 def _get_bundle(request, bundle_name):
     """
     Update bundle URLs to handle webpack hot reloading correctly if DEBUG=True
     """
     for chunk in get_loader('DEFAULT').get_bundle(bundle_name):
         chunk_copy = dict(chunk)
-        if settings.DEBUG and settings.USE_WEBPACK_DEV_SERVER:
-            chunk_copy['url'] = "{host_url}/{bundle}".format(
-                host_url=webpack_dev_server_url(request),
-                bundle=chunk['name'],
-            )
-        else:
-            chunk_copy['url'] = static("bundles/{bundle}".format(bundle=chunk['name']))
+        chunk_copy['url'] = "{host_url}/{bundle}".format(
```

Comment:
Yeah, it just takes care of the case where `host_url` contains a trailing slash already